### PR TITLE
Added capability to copy over other files

### PR DESCRIPTION
**Pull request by kdubovikov**
I have lots of images in my source directory. Added capability to copy over all non-md files

### CODE REVIEW

File: src/obsidian_to_org/__main__.py [hunk 1]
```diff
@@ -172,4 +194,4 @@ def convert_directory():
 
 
 if __name__ == "__main__":
-    main()
+    convert_directory()
```
Comment (jml) [inline src/obsidian_to_org/__main__.py:197]: **Question:** What's the motivation for this change? It seems unrelated to copying over images.

File: src/obsidian_to_org/__main__.py [hunk 2]
```diff
@@ -44,7 +45,9 @@ def fix_markdown_comments(markdown_contents):
 
 def restore_comments(org_contents):
     """Restore the comments in org format."""
-    return "".join(line.replace(COMMENT_MARKER, "# ") for line in  org_contents.splitlines(True))
+    return "".join(
+        line.replace(COMMENT_MARKER, "# ") for line in org_contents.splitlines(True)
+    )
```
Comment (jml) [inline src/obsidian_to_org/__main__.py:50]: **Question:** What's up with the formatting changes? I don't see any mention of them in the commits or in the PR description.

File: src/obsidian_to_org/__main__.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```
Comment (jml) [inline src/obsidian_to_org/__main__.py:1]: **Question:** What's the motivation for this change?

File: src/obsidian_to_org/__main__.py [hunk 4]
```diff
@@ -154,6 +171,11 @@ def convert_directory():
 
     for path in walk_directory(markdown_directory):
         if path.suffix != ".md":
+            copy_to = path.relative_to(markdown_directory)
+            copy_path = args.output_directory / copy_to
+            print(f"Copying from {path} to {copy_path}")
+            copy_path.parent.mkdir(parents=True, exist_ok=True)
+            shutil.copy(str(path), str(copy_path))
```
Comment (jml) [inline src/obsidian_to_org/__main__.py:178]: **Praise:** Thank you. This looks good to me.

**Review by jml — COMMENTED** [review]
Thanks! I'm glad this tool helped you. I have a couple of questions about some changes that seem unrelated.

File: src/obsidian_to_org/__main__.py (hunk 1 above)
Comment (kdubovikov) [reply src/obsidian_to_org/__main__.py:197]: Hi @jml . I didn't see `main` function in your script, so I have changed the script so that `convert_directory` would be called on start.

File: src/obsidian_to_org/__main__.py (hunk 2 above)
Comment (kdubovikov) [reply src/obsidian_to_org/__main__.py:50]: My editor formats files with `black` by default.

File: src/obsidian_to_org/__main__.py (hunk 3 above)
Comment (kdubovikov) [reply src/obsidian_to_org/__main__.py:1]: Please refer to this: https://stackoverflow.com/questions/43793040/how-does-usr-bin-env-work-in-a-linux-shebang-line. It makes the script more portable across different environments

File: src/obsidian_to_org/__main__.py (hunk 2 above)
Comment (jml) [reply src/obsidian_to_org/__main__.py:50]: Thanks! I thought mine did too, but apparently not. It seems I normally rely on pre-commit checks, which this repo doesn't have yet.

It would have made the review easier if the reformat was done in a separate commit to the logical changes.

File: src/obsidian_to_org/__main__.py (hunk 1 above)
Comment (jml) [reply src/obsidian_to_org/__main__.py:197]: That makes sense, thanks.
